mobify/progressive-web-scaffold
# Test fixes for projects

Adds the correct test paths to AVA, and installs jsdom for use by enzyme.mount. Also fixes the use of the fetch polyfill to match the project recommendations, and removes now-redundant lint rules.

Thanks to @cdegit for finding these.

## How to test-drive this PR
- npm install
- generate a component
- npm test
- see that the tests work and that the component tests run and pass
